test/search.spec.js

```diff
@@ -36,7 +36,7 @@ describe('Search', () => {
     let query = (temp.getMonth() + 1) + '-' + temp.getDate();
```

Comment:
### Critical

Use `let` only when
-  you are not initializing the variable on the same line
-  the value is going to be modified down the line

otherwise, use `const`